scripts/src/main/resources/scripts/command/aws

```diff
@@ -51,21 +64,19 @@ function doSetup() {
       AWS_VERSION=$(curl "https://github.com/aws/aws-cli/tags" | awk -F'tags/' '/archive/ { print $2}' | sort -r | head -1 | awk -F'.zip' '{print $1}')
     fi
 
-    doInstall "-" "${AWS_HOME}" "aws" "${AWS_VERSION}" 
+    doInstall "-" "${DEVON_IDE_HOME}/updates/install/aws" "aws" "${AWS_VERSION}" 
     if [ "${?}" = 0 ]
     then
-      doPackageInstall "${AWS_HOME}"
+      doPackageInstall "${DEVON_IDE_HOME}/updates/install/aws" "${AWS_VERSION}" 
     fi
-  fi
-
-  if [ "${1}" != "silent" ] && ! doIsQuiet
-  then
-    doEcho "logon again to test aws with \"aws --version\""
-  fi 
 
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      doEcho "logon again to test aws with \"aws --version\""
+    fi 
```

Comment:
I think this should stay outside the first if statement. Or is there a reason to move it inside?

Reply:
I will check it again